lint

```diff
@@ -94,10 +94,11 @@ lint_go() {
     # don't have it installed.  Also never blocks a commit,
     # it just warns.
     if type gocyclo >/dev/null 2>&1; then
-        gocyclo -over 25 "${filename}" | while read -r line; do
```

Comment:
Nit: `Do not spawn subshell`, we were just "_piping_" though, not spawning a subshell? (unless I missed something?)

Reply:
piping _does_ spawn a subshell

Reply:
For some reason I only thought of `sh [...]`/`bash [...]`, but you're right -- forget me! :sweat_smile: